index.js

```diff
@@ -95,14 +97,12 @@ Region.prototype.getOffset = function(x, z) {
   x = Math.abs(mod(x, 32))
   z = Math.abs(mod(z, 32))  
   locationOffset = 4 * (x + z * 32)
-  bytes = new Uint8Array(this.buffer, locationOffset, 4);
-  sectors = bytes[3];
-  offset = bytes[0] << 16 | bytes[1] << 8 | bytes[2];
-  if (offset === 0) {
-    return 0;
-  } else {
-    return offset * 4096;
-  }
+  this.dataView.seek(locationOffset);
```

Comment:
FWIW, the original `Uint8Array` code here works fine in node 0.10.4, 0.12.9, and 5.4.1